python/tops2vrt.py

```diff
@@ -92,115 +102,73 @@ def getLinePixelBbox(geobbox, latFile, lonFile):
-
-        tag = metadata['ACQUISITION_TIME'] 
-
-        vrttmpl='''<VRTDataset rasterXSize="{width}" rasterYSize="{height}">
-    <VRTRasterBand dataType="CFloat32" band="1" subClass="VRTRawRasterBand">
-        <sourceFilename>{PATH}</sourceFilename>
-        <ImageOffset>0</ImageOffset>
-        <PixelOffset>8</PixelOffset>
-        <LineOffset>{linewidth}</LineOffset>
-        <ByteOrder>LSB</ByteOrder>
-    </VRTRasterBand>
-</VRTDataset>'''
-        
-
-#        outname =  datetime.datetime.strptime(tag.upper(), '%d-%b-%Y %H:%M:%S UTC').strftime('%Y%m%d')
-
-        outname = metadata['ACQUISITION_TIME']
-        out_file = os.path.join(inps.outdir, '{0}.vrt'.format(outname))
-        print('{} / {}: {}'.format(ind+1, num_slc, out_file))
-        with open(out_file, 'w') as fid:
-            fid.write( vrttmpl.format(width=width,
-                                     height=height,
-                                     PATH=path,
-                                     linewidth=8*width))
-
-        data.append(metadata)
-        dates.append(outname)
-
-
-    ####Set up single stack file
-    if os.path.exists( inps.stackdir):
+    # Creating stack directory
+    if os.path.exists(inps.stackdir):
         print('stack directory: {0} already exists'.format(inps.stackdir))
     else:
         print('creating stack directory: {0}'.format(inps.stackdir))
-        os.makedirs(inps.stackdir)    
+        os.makedirs(inps.stackdir)
 
+    # Open full-res lat/lon arrays in radar geometry
     latFile = os.path.join(inps.indir, "geom_reference", "lat.rdr.full.vrt")
     lonFile = os.path.join(inps.indir, "geom_reference", "lon.rdr.full.vrt")
 
-    # setting up a subset of the stack
+    # Identify a subset area in the stack of co-registered SLCs
     if inps.geobbox:
         # if the bounding box in geo-coordinate is given, this has priority
-        print("finding bbox based on geo coordinates of {} ...".format(inps.geobbox))
-        ymin, ymax, xmin, xmax = getLinePixelBbox(inps.geobbox, latFile, lonFile)
-
+        print("finding bbox based on geo coordinates of {} ...".format(
+            inps.geobbox))
+        ymin, ymax, xmin, xmax = getLinePixelBbox(inps.geobbox, latFile,
+                                                  lonFile)
     elif inps.bbox:
         # if bbox in geo not given then look for line-pixel bbox
         print("using the input bbox based on line and pixel subset")
         ymin, ymax, xmin, xmax = inps.bbox
-
     else:
         # if no bbox provided, the take the full size
-        ymin, ymax, xmin, xmax = [0 , height, 0 , width]
+        ymin, ymax, xmin, xmax = [0, height, 0, width]
 
     xsize = xmax - xmin
     ysize = ymax - ymin
 
+    # Set-up filepath for VRT containing co-registered stack of SLCs
     slcs_base_file = os.path.join(inps.stackdir, 'slcs_base.vrt')
     print('write vrt file for stack directory')
+
+    # Allocate wavelength for Sentinel
+    wavelength = 0.05546576
```

Comment:
I know it has been there from before. Would you please make this an input argument default to Sentinel-1 wavelength.